README.md

```diff
@@ -101,9 +131,9 @@ You should now be able to view the site at http://127.0.0.1:8000/
 python manage.py test
 ```
 
-# HOWTO Contribute to this repo
+# How to contribute to this repo
 
-Refer to our contributing guidelines [here.](https://github.com/mercury-telemetry/mercury-telemetry/blob/master/CONTRIBUTING.md)
+Refer to our contributing guidelines [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/CONTRIBUTING.md)
```

Comment:
I think we should use the Contribute file in this new repo

Reply:
done